lib/api/liskApi.js

```diff
@@ -163,22 +163,35 @@ LiskAPI.prototype.sendRequest = function (requestType, options, callback) {
 	return privateApi.sendRequestPromise.call(this, requestType, options).then(function (requestSuccess) {
 		var returnAnswer = (parseOfflineRequest(requestType, options).requestMethod === 'GET') ? requestSuccess.body : parseOfflineRequest(requestType, options).transactionOutputAfter(requestSuccess.body);
 
-		if (!callback || (typeof callback !== 'function')) {
-			return Promise.resolve(returnAnswer);
-		} else {
-			return callback(returnAnswer);
+		if (callback && (typeof callback === 'function')) {
```

Comment:
Also,`LiskAPI.prototype.sendRequest` function is at the size when it might be worth to extract some part into a separate function.